### ISSUE TITLE
lovm ssh doesn't work with vagrant VMs

### ISSUE THREAD

**Issue by cbednarski** [cbednarski/lovm#1]
TL;DR vagrant's default NAT does not allow inbound SSH traffic so this doesn't work out of the box. There are two options:

1. Make SSH work by automatically forwarding a port for SSH (only)
2. Make SSH work by adding a second NIC to the VM that provides host-only networking. This is a lot more disruptive and requires coordination inside the guest OS to bring up the second NIC, but it allows all inbound ports to work (i.e. if you run a web server or database you would have to port forward each of these individually which is a huge pain)

I setup a proof-of-concept VM for this at some point and then I went to do something else but most of the notes are floating around somewhere.
